src/Pipedrive.net/Models/Request/ActivityUpdate.cs

```diff
@@ -10,15 +10,17 @@ public class ActivityUpdate
         [JsonProperty("subject")]
         public string Subject { get; set; }
 
+        [JsonProperty("public_description")]
+        public string PublicDescription { get; set; }
+
         [JsonProperty("done")]
         public ActivityDone Done { get; set; }
 
         [JsonProperty("type")]
         public string Type { get; set; }
 
         [JsonProperty("due_date")]
-        [JsonConverter(typeof(DateWithoutTimeConverter))]
-        public DateTime? DueDate { get; set; }
+        public string DueDate { get; set; }
```

Comment:
Open a specific issue for that. I'll take a look, don't change that here.

Reply:
Okay. Should I at least place a comment block here referencing the possible issue?